tools/test/spm/test_generate_partition_code.py

```diff
@@ -18,17 +18,13 @@
 import re
 import shutil
 import tempfile
-
-import jsonschema.exceptions as jexcep
 import pytest
+import jsonschema.exceptions as jexcep
 from jinja2.defaults import DEFAULT_FILTERS
-
-from .test_data import *
 from tools.psa.mbed_spm_tfm_common import *
-from tools.psa.generate_mbed_spm_partition_code import *
+from tools.psa.generate_partition_code import *
+from .test_data import *
```

Comment:
In general, we should stay away from `import *` to avoid bringing everything into the namespace. I realize in this case you're trying to do just that. For consistency with (most) of the tools, could you do one of the following:
- Import the module and call the functions on the module (this is usually my preferred choice).
    - If you're worries about line length, you can always do something like `import tools.psa.generate_partition_code as gpc`. This allows you to then call those functions with `gpc.my_func()`.
- Import each of the functions by name